dashboard/src/reducers/charts.ts

```diff
@@ -54,11 +55,16 @@ const chartsReducer = (
 ): IChartState => {
   switch (action.type) {
     case getType(actions.charts.requestCharts):
-      return { ...initialState, isFetching: true };
+      return { ...state, isFetching: true };
     case getType(actions.charts.requestChartsCategories):
       return { ...state, isFetching: true };
     case getType(actions.charts.receiveCharts):
-      return { ...state, isFetching: false, items: action.payload };
+      return {
+        ...state,
+        isFetching: false,
+        items: !action?.meta.length ? action.payload : state.items,
+        searchItems: action?.meta.length ? action.payload : state.searchItems,
```

Comment:
why are you checking `action?.meta.length`?

Reply:
This is the `query` value, just to know if the `receiveCharts` comes from a search query or not.